docs/content/mixin-dev-guide/distribution.md

```diff
@@ -68,4 +68,43 @@ unpublished builds from the master branch. The official Porter mixins follow
 this pattern. If you have other published tagged builds of your mixin, porter
 can handle installing them as well.
 
+## Search
+
+Porter maintains a list of remote mixins that users can search via
+`porter mixins search [NAME]`. If no name is supplied, the full listing will be
+returned.  See the help menu for all command options: `porter mixins search -h`.
+
+For example, here we search for a `cowsay` mixin:
+
+```console
+$ porter mixins search cowsay
+Name     Description                        Author           Source URL                                                    Feed URL
+cowsay   A mixin for using the cowsay cli   Porter Authors   https://github.com/deislabs/porter-cowsay/releases/download
+```
+
+## Broadcast
+
+To add your mixin to the list, add an entry to the remote mixin
+[list](https://github.com/deislabs/porter/blob/master/pkg/mixin/remote-mixins/index.json)
+with all the pertinent informational fields filled out.
+
+For instance, a new entry might look like:
+
+```json
+  {
+    "name": "mymixin",
+    "author": "My Name",
+    "description": "A mixin for doing great things",
+    "sourceURL": "https://github.com/org/project/releases/download",
+    "feedURL": "https://my.domain.com/mixins/atom.xml"
+  },
+```
+
+It is okay to leave `feedURL` empty if no atom feed yet exists.
```

Comment:
Ah, true, in CLI use, one supplies one or the other, not both.  Initially, I was thinking if a mixin actually had both options, both would be listed in its listing.  However, I can see this might be confusing.

Should we gravitate to only displaying `feedURL` or `sourceURL` (this latter to change to just `URL` as you've suggested elsewhere), with priority given to the `feedURL`, for a mixin listing?

Nevertheless, indeed, it would be more useful to document each field individually here.  I can do so.

Reply:
Right now, there aren't any mixins that are distributed by both GH releases and an atom feed. The intention was that really motivated people with multiple mixins would bother to make a feed (us) and people who make a single mixin would use GH releases. So it would end up being one or the other. 

If someone did happen to supply both, we should prefer the atom feed.